openedx/core/djangoapps/content_libraries/serializers.py

```diff
@@ -245,3 +250,60 @@ class ContentLibraryBlockImportTaskCreateSerializer(serializers.Serializer):
     """
 
     course_key = CourseKeyField()
+
+
+class ContentLibraryCollectionSerializer(serializers.ModelSerializer):
+    """
+    Serializer for a Content Library Collection
+    """
+
+    class Meta:
+        model = Collection
+        fields = '__all__'
+
+
+class ContentLibraryCollectionUpdateSerializer(serializers.Serializer):
+    """
+    Serializer for updating a Collection in a Content Library
+    """
+
+    title = serializers.CharField()
+    description = serializers.CharField()
```

Comment:
As commented in the other PR, I think we need something like this to allow updating the `key`:
```suggestion
    description = serializers.CharField()
    key = serializers.CharField()
```

Reply:
Nope, addressed here https://github.com/openedx/openedx-learning/pull/223#discussion_r1746402764